src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -0,0 +1,1100 @@
+import PhaseTapChangerPane from './phase-tap-changer-pane';
+import { useConnectivityValue } from '../connectivity-edition';
+import {
+    ActivePowerAdornment,
+    AmpereAdornment,
+    filledTextField,
+    MVAPowerAdornment,
+    OhmAdornment,
+    MicroSusceptanceAdornment,
+    VoltageAdornment,
+    sanitizeString,
+} from '../dialogUtils';
+import { REGULATION_MODES } from '../../network/constants';
+import { useBooleanValue } from '../inputs/boolean';
+import { EQUIPMENT_TYPE } from '@gridsuite/commons-ui';
+
+export const PHASE_TAP = 'dephasing';
+export const RATIO_TAP = 'ratio';
+
+/**
+ * Dialog to create a two windings transformer in the network
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param voltageLevelOptionsPromise Promise handling list of voltage level options
+ * @param currentNodeUuid : the node we are currently working on
+ * @param editData the data to edit
+ */
+const TwoWindingsTransformerCreationDialog = ({
+    editData,
+    open,
+    onClose,
+    voltageLevelOptionsPromise,
+    voltageLevelsEquipmentsOptionsPromise,
+    currentNodeUuid,
+}) => {
+    const studyUuid = decodeURIComponent(useParams().studyUuid);
+
+    const intlRef = useIntlRef();
+    const intl = useIntl();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const characteristicsInputForm = useInputForm();
+    const ratioTapInputForm = useInputForm();
+    const phaseTapInputForm = useInputForm();
+
+    const [formValues, setFormValues] = useState({});
+
+    const [ratioTapRows, setRatioTapRows] = useState([]);
+
+    const [phaseTapRows, setPhaseTapRows] = useState([]);
+
+    const [dialogWidth, setDialogWidth] = useState('sm');
+
+    const equipmentPath = '2-windings-transformers';
+
+    const [tabIndex, setTabIndex] = useState(0);
+
+    const clearValues = () => {
+        setFormValues(null);
+    };
+
+    const [isCopy, setIsCopy] = useState(false);
+
+    const [creationError, setCreationError] = useState(undefined);
+
+    /**
+     *
+     * CHARACTERISTICS TAP PANE
+     *
+     */
+
+    const [twoWindingsTransformerId, twoWindingsTransformerIdField] =
+        useTextValue({
+            label: 'ID',
+            validation: { isFieldRequired: true },
+            inputForm: characteristicsInputForm,
+            formProps: filledTextField,
+            defaultValue: formValues?.equipmentId,
+        });
+
+    const [twoWindingsTransformerName, twoWindingsTransformerNameField] =
+        useTextValue({
+            label: 'Name',
+            validation: { isFieldRequired: false },
+            inputForm: characteristicsInputForm,
+            formProps: filledTextField,
+            defaultValue: formValues?.equipmentName,
+        });
+
+    const [seriesResistance, seriesResistanceField] = useDoubleValue({
+        label: 'SeriesResistanceText',
+        validation: { isFieldRequired: true, isFieldNumeric: true },
```

Comment:
I think the isFieldNumeric is added by useDoubleValue

Reply:
True, i'm removing those